integration/spark/src/main/scala/org/apache/spark/sql/execution/datasources/SparkCarbonTableFormat.scala

```diff
@@ -126,6 +126,7 @@ with Serializable {
     optionsFinal.put(
       "fileheader",
       dataSchema.fields.map(_.name.toLowerCase).mkString(",") + "," + partitionStr)
+    optionsFinal.put("header", "false")
```

Comment:
please refer line 77 in org.apache.carbondata.processing.loading.model.CarbonLoadModelBuilder#build(java.util.Map<java.lang.String,java.lang.String>, long, java.lang.String), can be handled like that

Reply:
i think you can remove from filling in optionsLocal

Reply:
done